sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -1246,6 +1246,50 @@ class FileBasedDataSourceSuite extends QueryTest
       }
     }
   }
+
+  test("disable filter pushdown for collated strings") {
```

Comment:
The test case should be put in the loop so that all file sources are tested.

Reply:
the issue there is that some data sources still don't support collations (orc, parquet, csv) - should we maybe just test for parquet now, and as we add support for others we also add them to the test?

Reply:
oh, then this is fine to me.